README.md

```diff
@@ -35,20 +35,17 @@ To set up a local development-environment:
   To prevent conflicts between projects or components using other versions of Node.js it is recommended to use a 'Node version manager'.  
   Make sure to install the version specified in the [`.node-version`](.node-version)-file
 - Install Git (<https://git-scm.com/download/win>)
+- Clone the Git repository: `git clone https://github.com/global-121/121-platform.git`
 - Install Docker (<https://docs.docker.com/docker-for-windows/install/>)
 
 ### On Linux
 
 #### Install dependencies
-- Install Node.js  
-  To install the required version of Node.js and to prevent conflicts between projects or components using other versions of Node.js, use [NVM - Node Version Manager](http://nvm.sh/).  
-  After installing NVM run:
-
-      nvm install && nvm install-latest-npm
-
-- Install Git
-
-      sudo apt install git-all
+- Install Git: `sudo apt install git-all`
+- Clone the Git repository: `git clone https://github.com/global-121/121-platform.git`
+- Install Node.js: To install the required version of Node.js and to prevent conflicts between projects or components using other versions of Node.js, use [NVM - Node Version Manager](http://nvm.sh/).  
+  After installing NVM run: `nvm install-latest-npm`
+- `cd` to the directory of the cloned repository above and execute `nvm install`. In this way nvm "sees" which version of NodeJS needs to be installed.
```

Comment:
```suggestion
  After installing NVM run, from the root of this repository:  
  `nvm install && nvm install-latest-npm`
  So that the required version of Node.js and `npm` get installed.
```